spec/movie_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../classes/movie'
+
+movie = Movie.new('2020/2/2')
```

Comment:
- [ ] **[OPTIONAL]** By using `rspec hooks` you can define a local variable before or after each testing block, so whenever changes happen to the variable in that specific block of the test, it won't's effect the other block of the tests.

This is so helpful with complicated scenarios so for more information kindly check [Rspec - Hooks](https://www.tutorialspoint.com/rspec/rspec_hooks.htm)

eg: 

```rb
    before(:each) do
        @movie = Movie.new('2020/2/2')
    end
```